src/helpers.php

```diff
@@ -10,6 +10,13 @@ function modelsNamespace()
 if (!function_exists('factoriesNamespace')) {
     function factoriesNamespace()
     {
-        return config('poser.factories_namespace', config('poser.factories_directory', "Tests\\Factories\\"));
+        return config('poser.factories_namespace', "Tests\\Factories\\");
```

Comment:
This change would require a MAJOR version increase to 3.0.0. I'd like to avoid that if possible at the moment. Could we find a way around this, perhaps using a different name for the new `factories_directory` config entry?

Reply:
`poser.factories_directory` didn't actually exist before this PR, so I essentially just tidied `factoriesNamespace` here to the default expected response while adding the `poser.factories_directory`, since the case sensitivity would be an issue. I wouldn't expect this to be a breaking change at all.

Reply:
It used to: https://github.com/lukeraymonddowning/poser/releases/tag/2.0.1-beta 😛

We renamed it in this release and deprecated `poser.factories_directory`

Reply:
Ah yeah, makes sense now then. How's `poser.factories_location` instead? :man_shrugging:

Reply:
Sounds good to me 😀